packages/hiro-graph-client/src/eventstream.js

```diff
@@ -329,12 +329,25 @@ export default class EventStream {
                 );
 
                 if (this._scopeId) {
-                    ws.send(
-                        JSON.stringify({
-                            type: 'subscribe',
-                            id: this._scopeId,
-                        }),
-                    );
+                    if (typeof this._scopeId === 'string') {
+                        ws.send(
+                            JSON.stringify({
+                                type: 'subscribe',
+                                id: this._scopeId,
+                            }),
+                        );
+                    } else if (Array.isArray(this._scopeId)) {
+                        this._scopeId
+                            .filter((id) => Boolean(id))
```

Comment:
Do we need to make this filtering?

Reply:
🤔  we can pass something like this._scopeId = ["", "some-id"]

Reply:
we can pass also an object instead of a string, or invalid scopeId, or some other corrupted data: it's not the `client`'s responsibility. `Client` assumes that valid parameters were passed.

According to API, it might be a string or an array of strings - that's all, all other stuff should be handled outside the `client`

Reply:
ok, i've got it! then we do not need to filter. I will fix it, thanks!

Reply:
Reverted commit with unnecessary filtering